### ISSUE TITLE
git untracked files indicator not showing up with Meslo LG S for Powerline font

### ISSUE THREAD

**Issue by shikhanshu** [b-ryan/powerline-shell#267]
I am using Meslo LG S for Powerline font (size 11)

Everything works fine, except the git indicator for untracked files shows up as \U+2753, instead of the question mark that it should show up as.

Am I missing something?

**Comment by shikhanshu** [comment]
Tried pretty much all the Powerline fonts. None of them works.

**Comment by mness1978** [comment]
I just ran into this same issue after upgrading to the latest power line patched fonts.  I've been using DejaVuSansMono for years without a problem, but the \U+2753 code (BLACK QUESTION MARK ORNAMENT) doesn't seem to be implemented/patched anymore.

I temporarily solved this by patching the Unicode value in powerline_shell_base.py.  The list of characters to use is the first thing in the RepoStats class, around line 100 or so.  I changed the 'untracked' line to:

'untracked': u'\u003F',  #u'\u2753',

This replaces the stylized question mark with the standard question mark.
